config.go

```diff
@@ -285,6 +308,12 @@ func pluginCmd(path string) *exec.Cmd {
 		}
 	}
 
+	// No plugin binary found, so try to use an internal plugin.
+	if strings.Contains(path, command.TFSPACE) {
+		parts := strings.Split(path, command.TFSPACE)
+		return exec.Command(parts[0], parts[1:]...)
+	}
```

Comment:
Does this show up in the debug output? I'd like to make sure w/ `TF_LOG=trace` you can see where each plugin came from.